lib/SPIRVProducerPass.cpp

```diff
@@ -4514,6 +4677,24 @@ SPIRVID SPIRVProducerPassImpl::GenerateInstructionFromCall(CallInst *Call) {
     RID = addSPIRVInst(spv::OpFDiv, Ops);
     break;
   }
+  case Builtins::kDot:
+  case Builtins::kIDotAccSat:
+  case Builtins::kIDotAccSatPackedUUU:
+  case Builtins::kIDotAccSatPackedSSS:
+  case Builtins::kIDotAccSatPackedUSS:
+  case Builtins::kIDotAccSatPackedSUS:
+  case Builtins::kIDotPackedUUU:
+  case Builtins::kIDotPackedSSS:
+  case Builtins::kIDotPackedUSS:
+  case Builtins::kIDotPackedSUS:
+  case Builtins::kArmDotAcc: {
+    if (SpvVersion() >= SPIRVVersion::SPIRV_1_6) {
```

Comment:
It is supported via extension before SPIR-V 1.6. With 1.6 you no longer need to mention the extension.

Reply:
Done